File: MzLibMinecraft/src/main/java/mz/mzlib/minecraft/bukkit/entity/BukkitEntityUtil.java
package mz.mzlib.minecraft.bukkit.entity;

import mz.mzlib.minecraft.entity.Entity;

public class BukkitEntityUtil
{
    public static org.bukkit.entity.Entity toBukkit(Entity entity)
    {
        return EntityBukkit.create(entity.getWrapped()).getBukkitEntity().getWrapped();
    }

    public static Entity fromBukkit(org.bukkit.entity.Entity entity)
    {
        return CraftEntity.create(entity).getHandle();
    }
}

File: mzlib-minecraft/src/main/kotlin/mz/mzlib/minecraft/bukkit/entity/BukkitEntityUtil.kt
package mz.mzlib.minecraft.bukkit.entity

import mz.mzlib.minecraft.entity.Entity

val Entity.bukkit: org.bukkit.entity.Entity get() = BukkitEntityUtil.toBukkit(this)
val org.bukkit.entity.Entity.mz: Entity get() = BukkitEntityUtil.fromBukkit(this)
